prime.rb

```diff
@@ -0,0 +1,3 @@
+require 'prime'
+
+puts Prime.first(100)
```

Comment:
How could you hook this up to the web application part of the submission? Remembering also that the challenge was to produce the sum of primes up to a particular number...